packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -1,9 +1,21 @@
 import { pxToRem } from '../../../../lib'
-import { SiteVariablesPrepared, ColorSchemeMapping } from '../../../types'
-import { extendColorScheme } from '../../../colorUtils'
+import {
+  SiteVariablesPrepared,
+  ColorSchemeMapping,
+  ColorScheme,
+  ComponentAreaName,
+} from '../../../types'
+import { extendColorScheme, pickValuesFromColorScheme } from '../../../colorUtils'
+
+// TODO extract this to a util
+const tuple = <T extends string[]>(...args: T) => args
+const labelColorComponentAreasTuple = tuple<ComponentAreaName[]>('foreground', 'background')
```

Comment:
@kuzhelov we want to use here the `ComponentAreaName` values only, not just any string, that's why I've added the generic, but forgot to specify it. Sorry my bad. Is it more clear now?

Reply:
actually, got the intent - essentially, you would like to have strict string literal type in TS _being projected to run-time array_, now got it. Couple of things that we could simplify here:

```js
const tuple = <T extends string[]>(...args: T) => args
// to
const tuple = <T extends ComponentAreaName>(...args: T[]) => args
```

```js
const labelColorComponentAreasTuple = tuple<ComponentAreaName[]>('foreground', 'background')
// to
const labelColorComponentAreasTuple = tuple('foreground', 'background') // no need for explicit generic param
```

Reply:
also, we could rename `labelColorComponentAreasTuple ` to just `labelColorComponentAreas` - I think there is no need to specify type in the name, it will make things easier to read

Reply:
Changed, thanks for the suggestions